app/models/log_ticket.rb

```diff
@@ -0,0 +1,23 @@
+class LogTicket < ActiveRecord::Base
+  enum backend: [:s3, :local]
```

Comment:
Can we make this dependent on the environment, rather than the model/record? We shouldn't ever have both in any one environment. For development it would be pretty easy to flip back and forth if needed?